zerver/openapi/zulip.yaml

```diff
@@ -12125,6 +12202,54 @@ paths:
                 oneOf:
                   - $ref: "#/components/schemas/InvalidPushDeviceTokenError"
                   - $ref: "#/components/schemas/InvalidRemotePushDeviceTokenError"
+  /mobile_push/register:
+    post:
+      operationId: register-push-account
+      summary: Register push account for E2EE notifications
+      tags: ["mobile"]
+      description: |
+        Register an account on an install of the app to receive
+        end-to-end encrypted mobile push notifications.
+
+        **Changes**: New in Zulip 11.0 (feature level ZF-1653f2).
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                token_kind:
+                  description: |
+                    Whether the token was generated by FCM or APNs.
+
+                    - APNs = 1
+                    - FCM = 2
+                  type: integer
+                  example: 1
+                push_account_id:
+                  description: |
+                    The ID of the account on an install of the app to register.
+                  type: integer
+                  example: 9006
+                push_public_key:
+                  description: |
+                    The public key to use to encrypt the notifications.
```

Comment:
This should document how the client is supposed to generate this. You can likely copy some text from the spec.

Reply:
```
  Public part of the asymmetric key pair generated by the client
  using the NaCl (Networking and Cryptography Library), encoded
  using a RFC 4648 standard base64 encoder.

  It is used to encrypt notifications for delivery to the client.
```